jobs/base_model_validator.py

```diff
@@ -115,6 +115,28 @@ def process(self, input_model):
             yield errors.ModelMutatedDuringJobError(model)
 
 
+class ValidateCommitType(beam.DoFn):
+    """DoFn to check whether commit type is valid."""
+
+    def process(self, input_model):
+        """Function that defines how to process each element in a pipeline of
+        models.
+
+        Args:
+            input_model: datastore_services.Model. Entity to validate.
+
+        Yields:
+            ModelCommitTypeError. Error for commit_type validation.
+        """
+        model = jobs_utils.clone_model(input_model)
+        if not hasattr(model, 'commit_type'):
```

Comment:
Why do we need this? We can limit which types of models we pass in, shouldn't that be enough?

Reply:
Yes. I added it because I misunderstood and thought the validator should be placed called in class `BaseModelValidator`. I will update this in the next commit.

Reply:
Done.